source/includes/_modal_settings.md

```diff
@@ -100,3 +100,18 @@ window.wootricSettings = {
 <!--­­ end Wootric code --­­>
 ```
 Modal position can be either ```top``` or ```bottom``` (default).  To set the position, provide ```modal_position``` key-value in ```wootricSettings``` object.
+
+## Modal accessibility settings
+
+```js
+window.wootricSettings = {

```

Comment:
What do you think about adding an account token? in that way it will be easier for our users to follow up a proper installation path.